### PR TITLE
Initial Circle CI configuration (+ ESLint, editorconfig)

### PR DESCRIPTION
**Pull request by myabc**
Includes commits from https://github.com/OUP/rando/pull/7

---

### CODE REVIEW

**Comment by myabc** [comment]
We probably want a matrix build - in other words, to test against a couple different node versions. This should be possible, I just didn't get round to it yet :) e.g. https://github.com/facebook/jest/blob/master/.circleci/config.yml#L68

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+  root: true,
+
+  extends: ['@oupdev/mymaths-base'],
```
Comment (Nudelsieb) [inline .eslintrc.js]: How will this work? MyMaths is a private repo. We should probably find some global rules that we can derive from. Those can sit in a public repo. Maybe your `javascript` repo?

Comment (myabc) [reply .eslintrc.js]: @Nudelsieb this refers to `@oupdev/eslint-config-mymaths-base`, a public npm package that lives in a public repo:
https://github.com/OUP/javascript/tree/master/packages/eslint-config-mymaths-base

**Comment by myabc** [comment]
@Nudelsieb updated to test against multiple node versions

File: .eslintrc.js (hunk 1 above)
Comment (Nudelsieb) [reply .eslintrc.js]: Maybe we should consider renaming it because of wider scope this will be applied to, e.g. this project.

**Comment by mmrwoods** [comment]
@Nudelsieb Was this intentionally closed, or did you mean to merge it? Master is still broken on CI.

**Pull request merged**